node/tests/RedisClusterClient.test.ts

```diff
@@ -113,16 +138,15 @@ describe("RedisClusterClient", () => {
                     expect.not.stringContaining("# Errorstats"),
                 );
             });
-            client.close();
```

Comment:
how come in all tests we didn't have 'await' on closing? how did we missed that?

Reply:
🤷🏻‍♀️